### ISSUE TITLE
Accessibility backport: Image slider automatic transition setting and warning

### ISSUE THREAD

**Issue by makuhlmann** [shopware/platform#9504]
### Technical TODO

In Shopware 6.7, the automatic transition setting of the image slider has a big warning regarding accessibility issues attached to it, warning the user that it may break accessibility. Yet, the function can be enabled at the user's risk.

<details>
<summary>Screenshot</summary>

![Image](https://github.com/user-attachments/assets/43e33928-5ad5-4b58-9956-f69a793b9ad1)

</details>

In 6.6.10 that warning does not exist, and the feature cannot be enabled at all, citing accessibility issues. This is even the case when ACCESSIBILITY_TWEAKS are not enabled. Users that choose to ignore the accessibility requirements are prevented from enabling a feature that used to be available in previous releases of 6.6. A backport of the warning with the option to enable the function would restore that missing functionality.

<details>
<summary>Screenshot</summary>

![Image](https://github.com/user-attachments/assets/5ecb81c3-e881-4561-8943-6381364d2b29)

</details>
